theano/gpuarray/tests/test_basic_ops.py

```diff
@@ -396,7 +396,7 @@ def check(dtype, N, M_=None):
         k_symb = numpy.asarray(0)
         out = T.eye(N_symb, M_symb, k_symb, dtype=dtype)
         f = theano.function([N_symb, M_symb],
-                            out,
+                            T.stack(out),
```

Comment:
Oh, nevermind, I remember. Can you add a comment explaining that we add `stack` so that eye gets lifted to the GPU, since its inputs are not on the GPU, if that is actually the case?
Actually, I don't even know how that would help, since stack does not have any input on the GPU either... Can you check if it is still needed?